build.gradle

```diff
@@ -32,12 +21,18 @@ dependencies {
     }
 }
 
-task sourceJar(type: Jar) {
-    classifier = 'sources'
-    from sourceSets.main.allJava
-}
+version = "0.2.2"
 
-task groovydocJar(type: Jar, dependsOn: groovydoc) {
-    classifier = 'groovydoc'
-    from groovydoc.destinationDir
-}
\ No newline at end of file
+pluginBundle {
+    website = 'http://jbake.org/'
+    vcsUrl = 'https://github.com/jbake-org/jbake-gradle-plugin'
+    description = 'Adds the ability to run JBake directly from a Gradle build'
+    tags = ['jbake', 'site-generator']
+
+    plugins {
+        jbakePlugin {
+            id = 'me.champeau.jbake'
+            displayName = 'JBake Gradle Plugin'
+        }
+    }
+}
```

Comment:
I assumed it was left over from the old mechanism of publishing Gradle plugins. The _bintray.gradle_ is still there, so it can be incorporated again if you really want to publish to both locations.